manifests/inputs/kafkainput.pp

```diff
@@ -0,0 +1,151 @@
+# Input plugin that connects to a Kafka broker and subscribes to messages from the specified topic and partition
+#
+# === Parameters:
+#
+# $ensure::                      	This is used to set the status of the config file: present or absent
+#
+# $splitter::                    	Splitter to be used by the input. This should refer to the name of a
```

Comment:
Can you replace the commons parameters with a message as filepollinginput?
Meaning that from $splitter to $can_exit with this message:
   ### Common Input Parameters::    Check heka::inputs::tcpinput for the description
   #
Those parameters can change and it's easier to maintain the documentation in only one place